### PR TITLE
Added Arabic translate.

### PR DESCRIPTION
**Pull request by fawaz006**
I'm really didn't know how to build it and I also get error that say jsoncpp aren't found even it was installed, so I just coped fr.po and edit it, so I don't know if there's somthing wrong or all are ok

### CODE REVIEW

File: po/ar.po [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+msgid "Dark"
+msgstr "مظلم"
+
+#: src/ui/views/preferencesdialog.cpp:30
+msgid "Converter"
+msgstr "محول"
+
+#: src/ui/views/preferencesdialog.cpp:31
+msgid "Customize converter settings."
+msgstr "Customize converter settings."
```
Comment (nlogozzo) [inline po/ar.po:226]: I think you forgot to translate here.

**Review by nlogozzo — CHANGES_REQUESTED** [review]
Line 226

File: po/ar.po (hunk 1 above)
Comment (fawaz006) [reply po/ar.po]: wired, I remember that I translate it?

**Comment by nlogozzo** [comment]
LGTM, thanks!

**Pull request merged**